torch_xla_py/xla_dist.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+"""Tool to distribute training on Cloud TPU Pods."""
+from __future__ import division
+from __future__ import print_function
+
+import abc
+
+class BaseWorker(abc.ABC):
+
+  @classmethod
+  @abc.abstractmethod
+  def validate_cluster(cls, workers):
```

Comment:
A `worker` is a single entity. How can it, alone, validate a cluster (which assume a set of workers)?

Reply:
The idea was to have it as a `classmethod` which would be called on a set of workers. The reason I didn't use a `Cluster` and `validate` was because `Cluster` would only have a single method right now and not other purposes. But makes sense and have changed it to have a `Cluster` object.